tests/base.py

```diff
@@ -14,8 +14,8 @@ def setUp(self):
 
         # Set up paths to the test data directory and to the app templates directory
         root_dir = os.path.dirname(os.path.abspath(__file__))
-        self.data_path = str(os.path.join(root_dir, "data", ""))
-        self.templates_path = str(os.path.join(os.path.dirname(root_dir), "octue", "templates", ""))
```

Comment:
You know when you look at your own code from last year and you're like... "What was I on?"